.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: false
+
+language: go
+go:
+  - 1.8.x
+  - tip
+
+before_install:
```

Comment:
You'll need to add a 'go get -t -v ./...' to the before_install portion to fix your travis failure